src/guide/render-function.md

```diff
@@ -274,9 +274,9 @@ The `v-model` directive is expanded to `modelValue` and `onUpdate:modelValue` pr
 ```js
 props: ['modelValue'],
 render() {
-  return Vue.h('input', {
+  return Vue.h(SomeComponent, {
```

Comment:
I don't understand the need for this change.

Reply:
AFIK `modelValue` is handled in runtime and it is same for components and elements.

Reply:
I will try to expand a bit more on using the `v-model` directive in a render function. I changed `input` to `SomeComponent`, because `modelValue` simply will not work with HTML form elements. You have to either pass a regular `value` or use for example `vModelText` directive.

Reply:
My suggestion would be to merge this PR and expand on `v-model` usage in a render function in another PR, what do you think?